docs/README.md

```diff
@@ -37,9 +41,25 @@ The app records the request for notification and monitors inventory updates.  Wh
 
 ```
 
+5. Once you have added the `availability-notify` component, access your store's Admin.
+
+7. Go to **Customer** > **Message center** > **Templates**.
+
+9. Search for the `availability-notify` component template, named **BACK IN STOCK** and click on it.
+
+11. After, you will see the email template and its configuration. For example:
 
+
+![template-back-in-stock](https://user-images.githubusercontent.com/67270558/131547198-a4eb3f0e-5a20-4e63-9f1f-d3bb312fa621.gif)
 >ℹ️ The email template that will be used is `back-in-stock`
 
+
+Now you set up the template according to your necessities. Check out the next section, [Setting up the Back in stock template](#setting-up-the-back-in-stock-template), to learn how to do it.
+
+## Setting up the Back in stock template
```

Comment:
@polishq  just added this section to explain the settings of the template. Can you help with this part as well?

Reply:
Hmm, I'm not very experienced with that. Is there any general documentation on editing email templates that we could link to?

Reply:
There is the [Including order variables in email template](https://help.vtex.com/en/tracks/transactional-emails--6IkJwttMw5T84mlY9RifRP/fLMUCPArCYB9vcTZEZ6bi). I'm going to take a look at it.